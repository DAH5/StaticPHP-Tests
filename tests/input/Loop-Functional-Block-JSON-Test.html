--- loop( dir = "tests/input/loop", json = "tests/output/Loop-Functional-Block-JSON-Test.json" ) ---
<p><b>--- loop.title ---</b></p>
<p>--- loop.description ---</p>
<p><a href="--- loop.uri ---">Read More</a></p>
--- endloop ---