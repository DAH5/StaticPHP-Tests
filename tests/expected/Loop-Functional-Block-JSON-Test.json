[{"metadata":{"title":"Hello, world!","description":"This is a hello world file.","uri":"\/loop\/hello-world.html"},"fileContents":"<p>--- metadata.title ---<\/p>\r\n<p>--- metadata.description ---<\/p>","outputContent":"\r\n<p><b>Hello, world!<\/b><\/p>\r\n<p>This is a hello world file.<\/p>\r\n<p><a href=\"\/loop\/hello-world.html\">Read More<\/a><\/p>\r\n"},{"metadata":{"title":"Shopping List","description":"NOTE TO SELF: What to get when grocery shopping.","uri":"\/loop\/shopping-list.html"},"fileContents":"<ul>\r\n    <li>Bread<\/li>\r\n    <li>Butter<\/li>\r\n    <li>Milk<\/li>\r\n    <li>Water<\/li>\r\n<\/ul>","outputContent":"\r\n<p><b>Shopping List<\/b><\/p>\r\n<p>NOTE TO SELF: What to get when grocery shopping.<\/p>\r\n<p><a href=\"\/loop\/shopping-list.html\">Read More<\/a><\/p>\r\n"}]